src/apps/app-router.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * AppRouter controls which app is rendered based on the route.
+ * e.g.
+ * - /conversation/:conversationId renders Messenger
+ */
+
+import { Redirect, Route, Switch } from 'react-router-dom';
+import { MessengerMain } from '../messenger-main';
+import { FeatureFlag } from '../components/feature-flag';
+import { featureFlags } from '../lib/feature-flags';
+
+const redirectToRoot = () => <Redirect to={'/'} />;
+
+export const AppRouter = () => {
+  return (
+    <Switch>
```

Comment:
I'm using `react-router-dom` to decide which app to render. IIRC this was handled in state previously, but I can't remember how. Do we need to keep the selected app in state? If yes, I can work that out, but it currently doesn't seem like there's a need for it.